packages/tupaia-web/.eslintrc.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-env node */
+module.exports = {
+  extends: ['../../.eslintrc-ts-frontend.json'],
+};
```

Comment:
This has also snuck in. In this case I don't think it needs to be .gitignored, but just deleted on this branch.